new_platforms/vsextension/ProjectTemplates/oeenclave/MyEnclave.vcxproj

```diff
@@ -58,6 +70,12 @@
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" Label="Configuration">
+    <ConfigurationType>Makefile</ConfigurationType>
+    <PlatformToolset>v140</PlatformToolset>
+    <WindowsSDKDesktopARMSupport>true</WindowsSDKDesktopARMSupport>
```

Comment:
Aren't the `WindowsSDKDesktopARM*Support` bits only required when using the Windows SDK?

Reply:
Removed